### PR TITLE
8250840: some tests use --enable-preview unnecessarily

### PR DESCRIPTION
**Pull request by lahodaj**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8250840](https://bugs.openjdk.java.net/browse/JDK-8250840): some tests use --enable-preview unnecessarily

### Reviewers
 * [Vicente Romero](https://openjdk.java.net/census#vromero) (@vicente-romero-oracle - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/41/head:pull/41`
`$ git checkout pull/41`

### CODE REVIEW

File: test/langtools/tools/javac/ConditionalExpressionResolvePending.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
  *      jdk.compiler/com.sun.tools.javac.util
  * @build toolbox.ToolBox toolbox.JavacTask
  * @build combo.ComboTestHelper
- * @compile --enable-preview -source ${jdk.version} ConditionalExpressionResolvePending.java
+ * @compile ConditionalExpressionResolvePending.java
  * @run main/othervm --enable-preview ConditionalExpressionResolvePending
```
Comment (vicente-romero-oracle) [inline test/langtools/tools/javac/ConditionalExpressionResolvePending.java:38]: do you still need the @run main/othervm?

**Review by vicente-romero-oracle — APPROVED** [review]
just a minor change could be needed, there is no need for another round

File: test/langtools/tools/javac/ConditionalExpressionResolvePending.java (hunk 1 above)
Comment (lahodaj) [reply test/langtools/tools/javac/ConditionalExpressionResolvePending.java:38]: Yes, that's still needed. The test internally compiles some code with --enable-preview and then tries to load and run the code in a separate ClassLoader in the same VM. So --enable-preview at runtime is required.

Comment (vicente-romero-oracle) [reply test/langtools/tools/javac/ConditionalExpressionResolvePending.java:38]: ok looks good

**Pull request closed without merging**